updater/Gemfile.lock

```diff
@@ -289,7 +283,6 @@ GEM
       addressable (>= 2.8.0)
       crack (>= 0.3.2)
       hashdiff (>= 0.4.0, < 2.0.0)
-    zeitwerk (2.6.6)
```

Comment:
Most of this diff actually belongs in #6667, but somehow it was not introduced by that PR and our CI did not complain. Anyways, Dependabot is amending things for us :)